db/dcrsqlite/apisource.go

```diff
@@ -1110,20 +1110,43 @@ func (db *wiredDB) GetExplorerAddress(address string, count, offset int64) *expl
 		TotalUnspent: int64(totalreceived - totalsent),
 	}
 	return &explorer.AddressInfo{
-		Address:          address,
-		Limit:            count,
-		Offset:           offset,
-		Transactions:     addressTxs,
-		NumFundingTxns:   numFundingTxns,
-		KnownFundingTxns: numberMaxOfTx,
-		NumSpendingTxns:  numSpending,
-		NumUnconfirmed:   numUnconfirmed,
-		TotalReceived:    totalreceived,
-		TotalSent:        totalsent,
-		Unspent:          totalreceived - totalsent,
-		Balance:          balance,
+		Address:           address,
+		Limit:             count,
+		MaxLimit:          maxcount,
+		Offset:            offset,
+		Transactions:      addressTxs,
+		NumTransactions:   numTxns,
+		KnownTransactions: numberMaxOfTx,
+		KnownFundingTxns:  numReceiving,
+		NumSpendingTxns:   numSpending,
+		NumUnconfirmed:    numUnconfirmed,
+		TotalReceived:     totalreceived,
+		TotalSent:         totalsent,
+		Unspent:           totalreceived - totalsent,
+		Balance:           balance,
 	}
 }
 func ValidateNetworkAddress(address dcrutil.Address, p *chaincfg.Params) bool {
 	return address.IsForNet(p)
 }
+
+// SearchRawTransactionsForUnconfirmedTransactions returns the number of unconfirmed transactions,
+// given a maximum possible unconfirmed
+func (db *wiredDB) SearchRawTransactionsForUnconfirmedTransactions(address string, offset int64, maxUnconfirmedPossible int64) (numUnconfirmed int64, err error) {
```

Comment:
please rename to `CountUnconfirmedTransactions`.

At a glance, I would have expected more output from this function given the original name.  It calls SearchRawTransactionsVerbose, but that's just how the function gets the count, not the purpose of the function

Reply:
The `offset` argument does not make much sense here, unless you are planning to call the function repeatedly in the case where it returns `numUnconfirmed == maxUnconfirmedPossible`